packages/design/src/components/metrics-wheel.scss

```diff
@@ -0,0 +1,35 @@
+@use '../config.scss' as c;
+@use '../utilities/css-variable.scss' as cv;
+
+.#{c.$prefix}-metrics-wheel {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    svg {
+        width: 9rem;
+        height: 9rem;
+    }
+
+    &__circle {
+        stroke-width: 10;
+        stroke-linecap: round;
+        fill: transparent;
+        transition: stroke-dashoffset 0.5s ease-in-out;
+    }
+
+    &__label {
+        color: get-var('text-primary');
+        font-weight: bold;
+        text-anchor: middle;
+        dominant-baseline: middle;
+        font-size: 0.875rem;
+        letter-spacing: 0.16px;
+    }
+
+    @each $color in (blue, red, amber, orange, yellow, lime, green, emerald) {
```

Comment:
A todo for later is to make this into a variable with some comments, these types of _magic_ definitions are often prone to be missed when future changes are done.